client/src/components/map/layers/networks-markers/index.tsx

```diff
@@ -0,0 +1,208 @@
+import { useState, useMemo, useEffect } from 'react';
+
+import { Marker, useMap } from 'react-map-gl/maplibre';
+import Supercluster from 'supercluster';
+
+import { cn } from '@/lib/classnames';
+
+import { LayerProps } from '@/types/layers';
+
+import { OrganizationProperties, ProjectProperties, useMapNetworks } from '@/hooks/networks';
+
+import NetworksPopup, { PopupAttributes } from '@/components/map/networks-popup';
+
+export type NetworksLayerProps = LayerProps & {
+  beforeId?: string;
+};
+
+const sizes = {
+  '1-10': 'h-[40px] w-[20px]',
+  '10-50': 'h-[60px] w-[40px]',
+  '50-100': 'h-[80px] w-[60px]',
+  '>100': 'h-[100px] w-[80px]',
+};
+
+const getSize = (size: number) => {
+  if (size < 10) {
+    return sizes['1-10'];
+  }
+  if (size < 50) {
+    return sizes['10-50'];
+  }
+  if (size < 100) {
+    return sizes['50-100'];
+  }
+  return sizes['>100'];
+};
+
+type MarkerProps = {
+  id: string | number;
+  longitude: number;
+  latitude: number;
+  organizations: OrganizationProperties[];
+  projects: ProjectProperties[];
+  isCluster?: boolean;
+  onClick?: (type: 'project' | 'organization') => void;
+};
+
+const MarkerComponent = ({
+  id,
+  longitude,
+  latitude,
+  organizations,
+  projects,
+  isCluster = false,
+  onClick,
+}: MarkerProps) => (
+  <Marker key={`marker-${id}`} longitude={longitude} latitude={latitude}>
+    <div className="flex items-center gap-px">
+      {organizations.length > 0 && (
+        <button
+          type="button"
+          onClick={(e) => {
+            e.stopPropagation();
+            if (onClick) onClick('organization');
+          }}
+          className={cn(
+            'flex origin-left items-center justify-center bg-blue-500',
+            getSize(organizations?.length),
+            { 'border-2 border-black': !isCluster },
+          )}
+        >
+          <div className="text-sm text-white">{organizations?.length}</div>
```

Comment:
Format the number with something like this: https://github.com/Orcasa-Platform/orcasa-review4c/blob/b6b26ead65d33953ba0127cb604dd3c2fda25100/javascript/utils.js#L25-L30